sun-motion-simulator/src/Clock.jsx

```diff
@@ -32,16 +32,14 @@ export default class Clock extends React.Component {
     }
     render() {
         return <React.Fragment>
-            <div className="form-inline">
-                <label>
-                    The time of day:
-                    <input type="time"
-                           style={{width: '115px'}}
-                           value={this.displayTime(this.props.dateTime)}
-                           onChange={this.onTimeUpdate}
-                           className="form-control form-control-sm ml-2" />
-                </label>
-            </div>
+            <label>
+                The time of day:
```

Comment:
I noticed this wrapping the other day. This does fix the issue. Another option might be to rename this to: "Time"